tasks/Module_2/task_1.py

```diff
@@ -0,0 +1,17 @@
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+def display_inventory(inventory):
```

Comment:
According to Python Style Guide, you need should add two empty lines before and after a standalone function

Reply:
Could you add another empty line above?